src/xmpp/iq.c

```diff
@@ -93,6 +93,11 @@ typedef struct privilege_set_t {
     char *privilege;
 } ProfPrivilegeSet;
 
+typedef struct affiliation_list_t {
+    char *affiliation;
+    bool show;
```

Comment:
How about changing `show` to `handle_ui` or `show_ui_message` or similar? Something a bit more descriptive.